src/app/submission/pages/submission.component.ts

```diff
@@ -15,10 +15,10 @@ import {AlertService} from '@shared/services/alert.service';
 import {BrokerService} from '@shared/services/broker.service';
```

Comment:
This file is very big.
I would bet that some of the code belongs in a Service. For example, spreadsheet generation and download stuff.

Reply:
Extracted them to a new SpreadsheetTabDetailsComponent.